src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -704,6 +704,16 @@ void TwitchMessageBuilder::appendUsername()
         break;
     }
 
+    auto aliases = *getCSettings().aliasNames.readOnly().get();
+
+    for (auto const alias : aliases)
+    {
+        if (alias.getName().toLower() == this->message().loginName.toLower())
```

Comment:
Maybe storing the alias name as lowercase instead of converting it every time?

Reply:
I would assume the compiler would optimize it but I can explicitly write it as well.

Reply:
By storing I meant when saving the alias in the first place, but this is also fine 👍